web/src/routes/TuNewsDetailPage.tsx

```diff
@@ -100,12 +92,18 @@ const TuNewsDetailPage = ({ city, pathname, cityCode, languageCode }: CityRouteP
           <Svg src={TuNewsActiveIcon} width='100%' height='100%' />
         </IconContainer>
       </TuNewsBanner>
-      <Page
-        title={newsModel.title}
-        content={newsModel.content}
-        lastUpdate={newsModel.date}
-        showLastUpdateText={false}
-      />
+      {loading ? (
+        <SkeletonPage />
+      ) : (
+        newsModel && (
```

Comment:
🔧 The check for `loading` should not be necessary.
```suggestion
      {!newsModel ? (
        <SkeletonPage />
      ) : (
```